compiler/lib/src/main/scala/codegen/DictionaryJsonWriter/DictionaryJsonEncoder.scala

```diff
@@ -30,8 +30,12 @@ case class DictionaryJsonEncoder(
     dictionaryState: DictionaryJsonEncoderState
 ) {
     /** Dictionary entry map to JSON */
-    private def dictionaryEntryMapAsJson[A, B] (f1: (A, B) => Json) (map: Map[A, B]): Json =
-        (map.map { case (key, value) => f1(key, value) }).toList.asJson
+    private def dictionaryEntryMapAsJson[A <: BigInt, B] (f1: (A, B) => Json) (map: Map[A, B]): Json =
```

Comment:
Instead of a type parameter `A <: BigInt`, can we just use `BigInt`? Similarly for `String` in the next function. Also, instead of `String`, can we use `Name.Unqualified`? The second type is an alias of the first.

Reply:
This is resolved in my latest commit